features/adb-openshift.feature

```diff
@@ -0,0 +1,80 @@
+Feature: Command output from various OpenShift related commands in ADB
+  service-manager should return the correct output from commands affecting OpenShift in ADB
+
+  @ci
+  Scenario Outline: Boot and execute commands
+    Given box is <box>
+    And provider is <provider>
+    And a file named "Vagrantfile" with:
+    """
+    begin
+      require 'vagrant-libvirt'
+    rescue LoadError
+      # NOOP
+    end
+
+    Vagrant.configure('2') do |config|
+      config.vm.box = '<box>'
+      config.vm.box_url = 'file://../../.boxes/<box>-<provider>.box'
+      config.vm.network :private_network, ip: '<ip>'
+      config.vm.synced_folder '.', '/vagrant', disabled: true
+      config.servicemanager.services = 'docker'
+      config.vm.provision "shell", inline: <<-SHELL
+        systemctl enable openshift 2>&1
+        systemctl start openshift | true
```

Comment:
OOI, why the _| true_? This means the provisioning will always be "successful", even is starting the service failed, no?